src/main/webapp/app/quiz/manage/manage-buttons/quiz-exercise-manage-buttons.component.html

```diff
@@ -61,19 +65,24 @@
         }
     </div>
     @if (quizExercise.isAtLeastInstructor) {
+        @if (isDetailPage) {
+            <div class="{{ isDetailPage ? 'd-flex' : 'btn-group-vertical me-1 mb-1' }} ">
```

Comment:
always true here now

```suggestion
            <div class="d-flex">
```